addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyMissingScanRule.java

```diff
@@ -72,7 +69,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 
         // Content-Security-Policy is supported by Chrome 25+, Firefox 23+, Safari 7+, but not but
         // Internet Exploder
-        List<String> cspOptions = msg.getResponseHeader().getHeaderValues(HEADER_CSP);
+        List<String> cspOptions = msg.getResponseHeader().getHeaderValues(CspUtils.HEADER_CSP);
```

Comment:
I'd suggest using `HttpFieldsNames#CONTENT_SECURITY_POLICY` instead.

Reply:
Oh nice, I had totally forgot/missed that we added that.